From ad25b22e16bf2dacf0a157d0bfcb3c8d591d9055 Mon Sep 17 00:00:00 2001
From: Jeremy Bicha <jbicha@ubuntu.com>
Date: Wed, 31 Oct 2012 10:46:28 +0000
Subject: Show Printers & Region panels in Unity but hide Background

Ubuntu plans to fork a separate Appearance panel this cycle so
that Gnomebuntu can use the GNOME Background panel.

I believe Ubuntu will try to use the Printers and Region & Language
panels instead of system-config-printer & language-selector

https://bugzilla.gnome.org/show_bug.cgi?id=687258
---
diff --git a/panels/background/gnome-background-panel.desktop.in.in b/panels/background/gnome-background-panel.desktop.in.in
index b8f6bcc..268a7d3 100644
--- a/panels/background/gnome-background-panel.desktop.in.in
+++ b/panels/background/gnome-background-panel.desktop.in.in
@@ -7,7 +7,7 @@ Terminal=false
 Type=Application
 StartupNotify=true
 Categories=GNOME;GTK;Settings;DesktopSettings;X-GNOME-Settings-Panel;X-GNOME-PersonalSettings
-OnlyShowIn=GNOME;Unity;
+OnlyShowIn=GNOME;
 X-GNOME-Settings-Panel=background
 # Translators: those are keywords for the background control-center panel
 _Keywords=Wallpaper;Screen;Desktop;
